state-chain/pallets/cf-environment/src/lib.rs

```diff
@@ -447,9 +447,8 @@ pub mod pallet {
 		/// ## Errors
 		///
 		/// - [BadOrigin](frame_support::error::BadOrigin)
-		/// - [NonceAccountNotBeingUsedOrDoesntExist](Error::NonceAccountNotBeingUsedOrDoesntExist)
+		/// - [NonceAccountIsNotUnavailable](Error::NonceAccountNotBeingUsedOrDoesNotExist)
 		#[pallet::call_index(7)]
-		// Todo
 		#[pallet::weight(Weight::zero())]
 		pub fn update_sol_nonce(
```

Comment:
Just realised there's no benchmark for this...

Reply:
Can we please add one, or open an issue to add one (if it doesn't already exist)

Reply:
Just added it: https://linear.app/chainflip/issue/PRO-1556/solana-add-update-sol-nonce-benchmark